app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -30,5 +30,6 @@ module AuthnJwt
     OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
     VALID_CLAIM_NAME_REGEX = /^[a-zA-Z|$|_][a-zA-Z|$|_|0-9]*$/.freeze # starts with letter $ or _ can contains digits
     MANDATORY_CLAIMS_DENY_LIST = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME, JTI_CLAIM_NAME, AUD_CLAIM_NAME].freeze
+    MANDATORY_CLAIMS_CHARACTER_DELIMITER = ","
```

Comment:
```suggestion
    CLAIMS_CHARACTER_DELIMITER = ","
```
Mapping has the same delimiter between definitions

Reply:
done